packages/calypso-razorpay/src/index.tsx

```diff
@@ -0,0 +1,255 @@
+import debugFactory from 'debug';
+import { useRef, useEffect, useState, useContext, createContext, PropsWithChildren } from 'react';
+
+const debug = debugFactory( 'calypso-razorpay' );
+
+export interface RazorpayConfiguration {
+	js_url: string;
+	options: { key: string };
+}
+
+declare class Razorpay {
```

Comment:
Stripe.js used to be hot-loaded this way too before https://github.com/Automattic/wp-calypso/pull/56089. 

If you need later to explain what the `Razorpay` object can do, you can add things to this type. Here's what we used to do for Stripe: https://github.com/Automattic/wp-calypso/blob/c38fd60dadca86a0bb09d684682f4a964ab63ddc/packages/calypso-stripe/src/index.tsx#L44-L58